src/build/render/components/wrapper/wrapper.js

```diff
@@ -0,0 +1,145 @@
+const R = require("ramda");
+const {html, escapeHtml, REPO_URL, pageAnchor, localizer, detailsList} = require("../bits");
+const {renderMarkdown} = require("../markdown");
+const footer = require("./footer");
+const header = require("./header");
+const breadcrumbs = require("./breadcrumbs");
+const metabox = require("./metabox");
+const toc = require("./toc");
+const thanksList = require("./thanksList");
+
+const TOC_MIN_HEADERS = 2;
+const COLLAPSE_CHILD_PAGES = 8;
+const PREVIEW_LENGTH_CHARS = 100;
+
+const mainTopics = [
+  ["/h1/tags"],
+  ["/h1"]
+];
+
+const localizations = {
+  locale: {
+    en: "en_US",
+    es: "es_ES"
+  },
+  siteName: {
+    en: "The Reclaimers Library",
+    es: "La Biblioteca de Reclaimers"
+  },
+  toc: {
+    en: "On this page",
+    es: "En esta página"
+  },
+  children: {
+    en: "Child pages",
+    es: "Páginas secundarias"
+  },
+  related: {
+    en: "Related pages",
+    es: "Páginas relacionadas"
+  },
+  main: {
+    en: "Main topics",
+    es: "Temas principales"
+  },
+  edit: {
+    en: "Edit",
+    es: "Editar"
+  }
+};
+
+const langNames = {
+  en: "English",
+  es: "Español"
+};
+
+const wrapper = (ctx, headings, thanks, metaboxProps, body) => {
+  const {page, pageIndex, lang, buildOpts: {baseUrl}} = ctx;
+  const localize = localizer(localizations, lang);
+  const editPageUrl = `${REPO_URL}/edit/master/src/content${page.pageId}/readme${lang == "en" ? "" : "_" + lang}.md`;
+  const imgAbsoluteUrl = page.img ?
+    `${baseUrl}${page.localizedPaths[lang]}/${page.img}` :
+    `${baseUrl}/assets/librarian.png`;
+
+  const plaintextPreview = page._md ? `${renderMarkdown(page._md, pageIndex, true).substring(0, PREVIEW_LENGTH_CHARS)}...` : "";
+  const keywords = R.path(["keywords", lang], page);
+  const otherLangs = page.langs.filter(it => it != lang);
+
+  const thanksResult = thanksList(ctx, thanks);
+  if (thanksResult.headings) {
+    headings = [...headings, ...thanksResult.headings];
+  }
+
+  return html`
+    <!DOCTYPE html>
+    <html lang="${lang}">
```

Comment:
Language is specified here which tells browsers what language the page is in. I can also use it from CSS to apply language-specific styling.